lib/rodauth/features/verify_login_change.rb

```diff
@@ -29,6 +29,7 @@ module Rodauth
     auth_value_method :verify_login_change_login_column, :login
     session_key :verify_login_change_session_key, :verify_login_change_key
     auth_value_method :verify_login_change_table, :account_login_change_keys
+    translatable_method :verify_login_change_notice_flash, "An email has been sent to you with a link to verify your login change"
```

Comment:
I realise this clashes with the existing flash for when the verification takes place.

Do you have a preferred name for this would-be key? Perhaps `change_login_needs_verification_notice_flash`? It has the prefix for the `change_login` feature instead, but perhaps is clearer about its purpose (changing the message on _that_ feature's routes)?

Reply:
Since what I have currently proposed is definitely not workable, I will change it now to my suggestion above but will be happy to change it again if you have a preferred alternate.